ROX-19799: Update docs for VM2.0 (#78059)
openshift/openshift-docs: operating/manage-vulnerabilities/common-vuln-management-tasks.adoc

:_mod-docs-content-type: ASSEMBLY
[id="common-vuln-management-tasks"]
= Viewing and addressing vulnerabilities
include::modules/common-attributes.adoc[]
:context: common-vuln-management-tasks

toc::[]

[role="_abstract"]
Common vulnerability management tasks involve identifying and prioritizing vulnerabilities, remedying them, and monitoring for new threats.

> [id="viewing-vulnerabilities"]

Reviewer: _{context}??

The *Vulnerability Management* -> *Workload CVEs* page provides information about vulnerabilities in applications running on clusters in your system. You can view vulnerability information across images and deployments. The *Workload CVEs* page provides advanced filtering capabilities, including the ability to view images and deployments with vulnerabilities and filter by image, deployment, namespace, cluster, CVE, component, and component source.

//Viewing CVEs
include::modules/vulnerability-management20-view-cve.adoc[leveloffset=+1]

//Viewing vulns in nodes
include::modules/identify-vulnerabilities-in-nodes-vm20.adoc[leveloffset=+1]
include::modules/disable-identify-vulnerabilities-in-nodes.adoc[leveloffset=+2]
.Additional resources
* xref:../../operating/examine-images-for-vulnerabilities.adoc#supported-operating-systems_examine-images-for-vulnerabilities[Supported operating systems]

//viewing vulns in platforms

include::modules/vulnerability-management20-view-platform-cve.adoc[leveloffset=+1]

[id="excluding-CVEs"]
== Excluding CVEs

> You can exclude or ignore CVEs in {product-title-short} by snoozing node and platform CVEs and deferring or marking node, platform, and image CVEs as false positives. You might want to exclude CVEs if you know that the CVE is a false positive or you have already taken steps to mitigate the CVE. Snoozed CVEs do not appear in vulnerability reports or trigger policy violations.

Reviewer: ```suggestion
You can exclude or ignore CVEs in {product-title-short} based on snoozing node and platform CVEs and deferring or marking node, platform, and image CVEs as false positives. You need to exclude CVEs if you know that the CVE is a false positive or you have already taken steps to mitigate the CVE. Snoozed CVEs do not appear in vulnerability reports or trigger policy violations.
```

Author: This rewrite isn't accurate. They exclude or ignore CVEs by snoozing them or deferring them/marking as false positive. But they don't _have to_ exclude them if they know they are false positives.

> You can snooze a CVE to globally ignore it for a specified period of time. Snoozing a CVE does not require approval.

Reviewer: ```suggestion
You can snooze a CVE to ignore it globally for a certain period of time. Snoozing a CVE does not require approval.
```

Author: I think "specified" period of time is correct because you are choosing/specifying the time period. I don't see any benefit to using "certain" here.

[NOTE]
====

> Snoozing node and platform CVEs requires enabling the ROX_VULN_MGMT_LEGACY_SNOOZE environment variable.

Reviewer: ```suggestion
To snooze node and platform CVEs, you need to enable the `ROX_VULN_MGMT_LEGACY_SNOOZE` environment variable.
```

Author: The person snoozing CVEs is not necessarily the person enabling this feature, so I think it is more accurate to say that it requires the variable to be enabled.

====

> Deferring or marking a CVE as a false positive is done through the exception management workflow. This workflow provides the ability to view pending, approved, and denied deferral and false positive requests. You can scope the CVE exception to a single image, all tags for a single image, or globally across all images.

Reviewer: ```suggestion
Deferring or marking a CVE as false positive is done through the exception management workflow. This workflow provides the ability to view pending, approved and rejected deferral and false positive requests. You can restrict the CVE exception to a single image, all tags for a single image or globally for all images.
```

Author: The tab is literally called "denied requests," not rejected. Also, "scope" is accepted language here.

> When approving or denying a request, you must add a comment. A CVE remains in the observed status until the exception request is approved. A pending request for deferral that is denied by another user is still visible in reports, policy violations, and other places in the system, but is indicated by a *Pending exception* label next to the CVE when visiting *Vulnerability Management* -> *Workload CVEs*.

Reviewer: ```suggestion
If you approve or reject a request, you must add a comment. A CVE remains in the observed status until the exception request is approved. A pending deferral request that is rejected by another user is still visible in reports, policy violations and elsewhere in the system, but is indicated by a *Pending exception* label next to the CVE when you click *Vulnerability Management* -> *Workload CVEs*.
```

Author: The language here is specific because requests are denied (not rejected). Also after "policy violations" the Oxford comma is necessary per IBM Style. I don't think "when you click" is needed to replace "visiting" since the idea is that anyone going to this screen will see the labels.
